docs/signed-apk-android.md

```diff
@@ -38,7 +38,8 @@ Navigate to that directory by using the command `cd /your/jdk/path` and use the
 ## Setting up Gradle variables
 
 1. Place the `my-upload-key.keystore` file under the `android/app` directory in your project folder.
-2. Edit the file `~/.gradle/gradle.properties` or `android/gradle.properties`, and add the following (replace `*****` with the correct keystore password, alias and key password),
+2. Edit the file `~/.gradle/gradle.properties` or `android/gradle.properties`, and add the following
+3. Replace `*****` with the correct keystore (line 3) and key (line 4) passwords, if you were not prompted to create a key password it will be the same as keystore password
```

Comment:
Let's not add a new bullet point here. I don't think it's necessary to specify `line 3` and `line 4` also.

Reply:
>Let's not add a new bullet point here.

Do you have a better suggestion to make it clear for people using this doc?

>I don't think it's necessary to specify line 3 and line 4 also.

One of the configs is called `storePassword` and the other is called `keyPassword`. Someone reading this documentation for the first time will be confused because the doc mention "keystore password" (confusingly the combination of both names). I am familiar with the term, and I was questioning myself about the correct place to use the variable. Because of that, I believe one extra level of clarification is needed.

Reply:
I've left two suggestions that clarifies a bit more.